README.md

```diff
@@ -1,69 +1,60 @@
-|**compiler**       | [GCC](http://gcc.gnu.org)                            | ≥ 7      | no other compiler is currently supported! |
-|**build system**   | [cmake](https://cmake.org)                           | ≥ 3.4    | optional, but recommended                 |
-|**required libs**  | [SDSL](https://github.com/xxsds/sdsl-lite)           | ≥ 3      | succint datastructures                    |
-|                   | [Ranges-V3](https://github.com/ericniebler/range-v3) | == 0.3.* | ranges and views                          |
-|**optional libs**  | [Cereal](https://github.com/USCiLab/cereal)          | ≥ 1.2.3  | serialisation                             |
-|                   | [Lemon](http://lemon.cs.elte.hu)                     | ≥ 1.3.1  | graphs, required for MSA                  |
+## Quick facts
 
-### Developers of the library
+  * C++ header-only library: easy to integrate with your app & easy to distribute
+  * liberal open source license: allows integration with any app or library, requires only attribution
+  * very high code quality standards: >97% unit test coverage, performance regression tests...
+  * extensive API documentation & tutorials: more lines of documentation than lines of code
 
-To build the tests and API documentation, you also need:
+## Dependencies
 
-|                   | requirement                                          | version  |
-|-------------------|------------------------------------------------------|----------|
-|**build system**   | [cmake](https://cmake.org)                           | ≥ 3.4    |
-|**test system**    | [GoogleTest](https://github.com/google/googletest)   | ≥ 1.8    |
-|**doc system**     | [Doxygen](https://github.com/doxygen/doxygen)        | ≥ 1.8    |
+|                   | requirement                                          | version  | comment                                    |
+|-------------------|------------------------------------------------------|----------|--------------------------------------------|
+|**compiler**       | [GCC](http://gcc.gnu.org)                            | ≥ 7      | no other compiler is currently supported!  |
+|**build system**   | [cmake](https://cmake.org)                           | ≥ 3.4    | optional, but recommended                  |
+|**required libs**  | [SDSL](https://github.com/xxsds/sdsl-lite)           | ≥ 3      |                                            |
+|                   | [Range-V3](https://github.com/ericniebler/range-v3)  | ≥ 1.0    |                                            |
+|**optional libs**  | [Cereal](https://github.com/USCiLab/cereal)          | ≥ 1.2.3  | required for serialisation and CTD support |
+|                   | [zlib](https://github.com/madler/zlib)               | ≥ 1.2    | required for `*.gz` and`.bam` file support |
+|                   | [bzip2](http://www.bzip2.org)                        | ≥ 1.0    | required for `*.bz2` file support          |
```

Comment:
```suggestion
|                   | [bzip2](http://www.bzip.org)                        | ≥ 1.0    | required for `*.bz2` file support          |
```

The provided link doesn't work.

Reply:
Maybe there is also a more helpful link. There isn't too much on the bzip.org website, and the download section didn't work for me. But it seems to be the official website.

Reply:
Yeah, it's not so important. All unixes have it anyway. I jsut want to give some global reference.